src/main/java/com/targomo/client/api/geo/AbstractLocation.java

```diff
@@ -0,0 +1,47 @@
+package com.targomo.client.api.geo;
+
+import com.targomo.client.api.pojo.AggregationInputParameters;
+
+public abstract class AbstractLocation {
+
+    protected String id;
+    protected AggregationInputParameters aggregationInputParameters;
```

Comment:
adding this to the abstract location could be a little confusing tbh. in multigraph AggregationInputParameters are at a different location. We might have to look for an alternative way although I don't really have an idea how we could do it better. We probably should have a quick brainstorming session or chat about how we model this here.

Reply:
hum ok how and when should we chat about that?